apps/swirl-docs/src/components/Layout/CategoryNav.tsx

```diff
@@ -35,22 +32,20 @@ export const CategoryNav: FunctionComponent<CategoryNavProps> = ({
   // );
 
   const SubElement = ({ navItem }: { navItem: NavItem }) => (
-    <NoSsr>
-      <li
-        className={`font-sm mb-4 ${
-          activePath === navItem.url ? "text-border-info" : "text-text-default"
-        }`}
-      >
-        <Link href={`${navItem.url}`}>
-          <a className=" text-sm">{capitalizeFirstLetter(navItem.title)}</a>
-        </Link>
-      </li>
-    </NoSsr>
+    <li
+      className={`font-sm mb-4 ${
+        activePath === navItem.url ? "text-border-info" : "text-text-default"
+      }`}
+    >
+      <Link href={`${navItem.url}`}>
+        <a className="text-sm capitalize">{navItem.title}</a>
+      </Link>
+    </li>
   );
 
   return (
     <nav
-      aria-label="category"
+      aria-label={ariaLabel ? ariaLabel : "Category Navigation"}
```

Comment:
**issue:** 'Category' instead of 'Category Navigation'. The 'Navigation' part is implied through the `nav` element.